src/test/resources/com/scriptbasic/testprograms/TestIf2.bas

```diff
@@ -0,0 +1,8 @@
+'
```

Comment:
we need much more complex test cases, like

```
if true then if true then print "a" else print "b" else print "c"
```

Reply:
The single-line 'IF' statement is very dangerous because it does not address properly the dangling ELSE problem. There can be many different cases. One command between the 'THEN' and 'ELSE', many commands, empty commands.

```
IF x THEN a=1 :ELSE: IF y THEN: a=2 ELSE: a=3
```
This is the main reason that one line IF was not implemented in ScriptBasic. There was no aim to be compatible with RealBASIC or any other basic.

I accept compatibility with any BASIC as that increases usability and after all the ultimate goal is that ppl use the program. Still it feels a bit itchy.

Reply:
I tested more complex single-line 'IF' statements and fixed one more issue. There are two tests (TestIf3.bas and TestIf4.bas) to verify these cases. I tested same IF statements in Visual Basic with same results.